api/groth16_messages.proto

```diff
@@ -1,17 +1,17 @@
 syntax = "proto3";
 
-package proverpkg;
+package prover_proto;
 
 import public "api/util.proto";
 
-message R1csGgPpzksnarkVerificationKey {
+message VerificationKeyGROTH16 {
     HexadecimalPointBaseGroup1Affine alpha_g1 = 1;
```

Comment:
Just a thought, but while doing this big rename, how about renaming these to `HexPoint....`?  It shortens things without adding any ambiguity.
Also, out of curiosity, what does `Base` refer to here?

Reply:
`HexPoint....` that works for me yes.

> Also, out of curiosity, what does Base refer to here?
A bilin. pairing is denoted as e: G1 x G2 -> Gt, where both G1 and G2 are the base groups, and where Gt is the target group